content/SCALE/SCALETutorials/Virtualization/CreatingManagingVMsSCALE.md

```diff
@@ -18,45 +18,123 @@ VMs provide more isolation than Jails but also consumes more system resources.
+
+**Minimum Memory Size** is an optional field. When not specified, the guest system is given the fixed amount of memory specified in **Memory Size (Examples: 500KiB, 500M, 2 TB)**. When **Minimum Memory Size** is specified, the guest system is given memory within the range **Minimum Memory Size** and **Memory Size (Examples: 500KiB, 500M, 2 TB)** as needed.
+
+The next field, **Optional: NUMA nodeset (Example: 0-1)** is also not populated by default as it is not required. If you specified CPU set you can then specify a NUMA nodeset to improve memory locally.
+
+Click **Next**.
+{{< /expand >}}
+
+### Configuring Disks
+{{< expand "Click Here for More Information" "v" >}}
+![CreateVMWDisksSCALE](/images/SCALE/22.12/CreateVMWDisksSCALE.png "VM Disks")
+
+Select **Create new disk image** radio button to create a new zvol on an existing dataset (you define the location of this existing dataset in **Zvol Location**).  Alternatively, you can select **Use existing disk image** if there is an existing zvol you want to use for the VM.
+
+The dropdown list under **Select Disk Type** allows you  to select either **AHCI** or **VirtIO**. We recommend using **AHCI** as the **Disk Type** for Windows VMs. Next, specify the location of the dataset in **Zvol Location**. Here we have specified the pool called *tank*. Enter a value in **Size (Examples: 500KiB, 500M and 2TB)** to indicate the amount of space to allocate for the new zvol.
+
+Click **Next**.
+{{< /expand >}}
+
+### Configuring the Network Interface
+{{< expand "Click Here for More Information" "v" >}}
+![CreateVMWNetworkInterfaceSCALE](/images/SCALE/22.12/CreateVMWNetworkInterfaceSCALE.png "VM Network Interface")
 
-For more information see [Virtualization Screens]({{< relref "VirtualizationScreens.md" >}}) for more information on virtual machine screen settings.
+Under **Adapter Type**, select **Intel e82585 (e1000)** from the dropdown list as it offers the higher level of compatiblity with most operating systems. Select **VirtIO** if the guest operating system supports para-virtualized network drivers.
+
+A randomized MAC address displays in the **Mac Address** field. You can change this to suit your needs.
+
+Use the **Attach NIC** dropdown list to select the active interface you wish to use. Active interfaces can be viewed at **Network** > **Interfaces** (note that if you navigate away from the wizard at this point you will lose your progress).
+
+The **Trust Guest Filters** checkbox is not selected by default. Enabling this feature has security risks, because it allows the virtual server to change its MAC address and so receive all frames delivered to this address. For more information see [Virtualization Screens]({{< relref "VirtualizationScreens.md#network-interface-screen" >}}).
```

Comment:
```suggestion
The **Trust Guest Filters** checkbox is not selected by default. Enabling this feature has security risks because it allows the virtual server to change its MAC address and so receive all frames delivered to this address. For more information see [Virtualization Screens]({{< relref "VirtualizationScreens.md#network-interface-screen" >}}).
```

Reply:
Change made.